### PR TITLE
Set up CI

### PR DESCRIPTION
**Pull request by peterstuart**

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```
Comment (peterstuart) [inline .github/workflows/nodejs.yml]: I stripped this down from the default node workflow which builds on 10.x and 12.x. I'm not really familiar with differences between node versions. Do you think it's worth testing on both?

File: src/addRef.coffee [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 import {useRef} from 'react'
 
+import {isFunction} from './util/helpers'
+
 addRef = (name, initialValue) ->
   (props) ->
-    ref = useRef (if isFunction initialValue then initialValue props else initialValue)
+    ref = useRef(
+      if isFunction initialValue then initialValue props else initialValue
+    )
```
Comment (peterstuart) [inline src/addRef.coffee:9]: Whoops, guess I didn't run `yarn lint` on my last PR.

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (helixbass) [reply .github/workflows/nodejs.yml]: I don't know a ton about what might break against different Node versions, so I'd probably stick with the default

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: yarn install
+      run: yarn install
+    - name: yarn lint
+      run: yarn lint
+    - name: yarn build
+      run: yarn build
+    - name: yarn text
```
Comment (helixbass) [inline .github/workflows/nodejs.yml]: `text` -> `test`

**Review by helixbass — APPROVED** [review]
@peterstuart looks good, see a couple comments

File: .github/workflows/nodejs.yml (hunk 3 above)
Comment (peterstuart) [reply .github/workflows/nodejs.yml]: 👍

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (peterstuart) [reply .github/workflows/nodejs.yml]: 👍

**Pull request merged**